src/_init.js

```diff
@@ -55,9 +55,10 @@ module.exports = function _init(arc) {
     },
     update(params, callback) {
       params.TableName = TableName
-      _doc.update(params, function _update(err) {
+      _doc.update(params, function _update(err, results) {
+        console.log('RESULTS:', results)
```

Comment:
lets remove the log! (idk whats up w the builds…investigating)

Reply:
D'oh!  Done.  I wonder if the console.log borked the build.